src/main/java/com/bobocode/nasapicturesstealer/persistance/entity/Picture.java

```diff
@@ -0,0 +1,42 @@
+package com.bobocode.nasapicturesstealer.persistance.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import static jakarta.persistence.GenerationType.IDENTITY;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+@Table(name = "pictures", schema = "public")
+public final class Picture {
+
+    @Id
+    @GeneratedValue(strategy = IDENTITY)
+    @Column(name = "id")
+    private long id;
+
+    @Column(name = "nasa_id")
+    private int nasaId;
+
+    @Column(name = "img_src")
+    private String imgSrc;
+
+    @ManyToOne
+    @JoinColumn(name = "camera_id")
```

Comment:
Exclude from equals and hash code

Reply:
so in this case, should I implement equals and hashcode?